lib/model-manager.js

```diff
@@ -4,26 +4,21 @@ const Toposort = require('toposort-class');
 const _ = require('lodash');
 
 class ModelManager {
-  constructor(sequelize) {
+  constructor() {
     this.models = [];
-    this.sequelize = sequelize;
   }
 
   addModel(model) {
     this.models.push(model);
-    this.sequelize.models[model.name] = model;
-
     return model;
   }
 
   removeModel(modelToRemove) {
     this.models = this.models.filter(model => model.name !== modelToRemove.name);
-
-    delete this.sequelize.models[modelToRemove.name];
   }
 
-  getModel(against, options) {
-    options = _.defaults(options || {}, {
+  getModel(against, options = {}) {
```

Comment:
fyi there is a difference, if you passed anything `falsy` as `options` it would default to empty object, now it will only if the value is not specified.
This change is _probably_ fine though.